src/Microsoft.VisualStudio.Web.CodeGeneration.Design/Program.cs

```diff
@@ -41,6 +41,10 @@ private static void Execute(string[] args, ConsoleLogger logger)
             var noBuild = app.Option("--no-build", "", CommandOptionType.NoValue);
             var simMode = app.Option("--simulation-mode", Resources.SimulationModeOptionDesc, CommandOptionType.NoValue);
 
+            while (!System.Diagnostics.Debugger.IsAttached)
+            {
+            }
+
```

Comment:
On second thought, it might not be such a bad idea to put this under a `#if DEBUG` block conditionally if a flag like `--debug` is passed in.